secure/README.md

```diff
@@ -0,0 +1,34 @@
+# Securing Dataproc
+
+This initialization action installs the MIT distribution of the Kerberos packages, and configure Yarn, HDFS, Hive, Spark to integrate with Kerberose, which enables in-transit data encryption.
+It will also encrypt HTTP traffic (include Web UIs and shuffle) with SSL.
+
+## Using this initialization action
+You can use this initialization action to create a secured Dataproc cluster:
+  Using the `gcloud` command to create a new cluster with this initialization action.
+
+```bash
+gcloud dataproc clusters create <CLUSTER_NAME> \
+    --scopes cloud-platform \
```

Comment:
Why is this needed?

Reply:
The default Dataproc scope is not enough since we need to also use KMS for password decryption. For example simplicity I just used the 'cloud-platform' scope.